src/components/Panel/Panel.js

```diff
@@ -29,6 +46,35 @@ class Panel extends Component {
           />
         )}
 
+        {collapsible && (
+          <button
+            className={cssClasses.collapseButton}
+            aria-expanded={!this.state.collapsed}
+            onClick={event => {
+              event.preventDefault();
+
+              this.setState(previousState => ({
+                controlled: true,
+                collapsed: !previousState.collapsed,
+              }));
+            }}
+          >
+            {this.state.collapsed ? (
+              <Template
+                {...templateProps}
+                templateKey="collapseButtonMore"
```

Comment:
Why two templates rather than one with the argument `collapsed`? Like we do inside the [`showMore`](https://www.algolia.com/doc/api-reference/widgets/refinement-list/js/?language=js#widget-param-showmoretext) template.

Reply:
I forgot we went for this. Updating!